src/CosmosEditorManager.ts

```diff
@@ -45,7 +45,7 @@ export class CosmosEditorManager {
     private fileMap: { [key: string]: ICosmosEditor } = {};
     private ignoreSave: boolean = false;
 
-    private readonly showSavePromptKey: string = 'cosmosDB.showSavePrompt';
+    private readonly showSavePromptKey: string = 'azureDatabases.showSavePrompt';
```

Comment:
A big part of why I'm having the discussion in https://github.com/microsoft/vscode-cosmosdb/pull/1442 is because of this setting. If we only use the editor for Cosmos DB and not PostgreSQL, I feel like we can keep this setting as-is and avoid people seeing the prompt after they've already clicked "Don't show again"

You could argue that now is a good time to break people (since we're doing all the re-branding anyways) in case other database types ever use the editor manager, but I feel like it might be possible we switch to the virtual file system provider before another database type ever needs this setting